xray/manager/xray.go

```diff
@@ -197,3 +187,28 @@ func (sm *XrayServicesManager) IsEntitled(featureId string) (bool, error) {
 	entitlementsService.XrayDetails = sm.config.GetServiceDetails()
 	return entitlementsService.IsEntitled(featureId)
 }
+
+// IsXscEnabled will try to get XSC version. If route is not available, user is not entitled for XSC.
+func (sm *XrayServicesManager) IsXscEnabled() (xscEntitled bool, xsxVersion string, err error) {
```

Comment:
```suggestion
func (sm *XrayServicesManager) IsXscEnabled() (xsxVersion string, err error) {
```

no need to have boolean and version response
xsc is enabled when version is not empty